docs/src/quickstart.md

```diff
@@ -307,12 +306,12 @@ CloudNativePG provides a default dashboard for Grafana as part of the official
 [grafana-dashboard.json](https://github.com/cloudnative-pg/grafana-dashboards/blob/main/charts/cluster/grafana-dashboard.json)
 file and manually importing it via the GUI.
 
-!!! Warning
+> [!WARNING]
     Some graphs in the previous dashboard make use of metrics that are in alpha stage by the time
     this was created, like `kubelet_volume_stats_available_bytes` and `kubelet_volume_stats_capacity_bytes`
     producing some graphs to show `No data`.
 
 ![local grafana](images/grafana-local.png)
 
-Note that in our example setup, both Prometheus and Grafana will pick up
-any other CloudNativePG clusters deployed with Monitoring activated.
+> [!NOTE] 
```

Comment:
If you mean to have this as a boxed aside, please use the !!! admonition format.
But ... is this worth and admonition box?

Reply:
That was my error i didn't know that it's not compatible.

Reply:
@jsilvela I actually did not know about admonition format. I did some research and managed to update the requested changes and created a new commit. 🙏Please let me know if there is anything else you'd like for me to update.